src/nidigital/examples/nidigital_ppmu_source_and_measure.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import pytest
+import sys
+import time
+
+
+def example(resource_name, options, channels, measure, aperture_time,
+            command, settling_time=None, current_level_range=None, current_level=None,
```

Comment:
Rename `command` to `source` for better readability

Reply:
Done